src/main/java/org/terasology/MineSweeper/system/MinesweeperSystem.java

```diff
@@ -15,14 +15,13 @@
  */
 package org.terasology.MineSweeper.system;
 
+import com.google.common.collect.Maps;
 import com.google.common.collect.Queues;
 import com.google.common.collect.Sets;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.terasology.MineSweeper.blocks.SweeperFamilyUpdate;
-import org.terasology.MineSweeper.component.CountComponent;
-import org.terasology.MineSweeper.component.MineComponent;
-import org.terasology.MineSweeper.component.FloatingCountComponent;
+import org.terasology.MineSweeper.component.*;
```

Comment:
Please avoid star imports. There is a setting in IntelliJ where you can set the minimum numbers of package members before collapsing to a star import - just set that really high like 999 and it should auto-format away.

Reply:
I only put that one as a star because it needs to import every class in the package, but I'll change it back.